### DISCUSSION TITLE
No Touch on adafruit 3.5" 480*320 TFT FeatherWing V2 (TSC2007)?

### DISCUSSION THREAD

**Discussion by TheRealSimon42** [HASwitchPlate/openHASP#748] [Q&A]
Hello everyone,
I am just trying to get started with this wonderful Project here, but run into the issue that my Touchscreen is not working. The screen itself is lighting up and showing everything correct, but the touch is not working (I am just able to navigate via the website, when enabling the cursor it stays where it is).

Hardware: 

- [Adafruit HUZZAH32 Assembled Plain Headers](https://exp-tech.de/en/products/adafruit-huzzah32-assembled-plain-headers?variant=47551050678599)
- [Adafruit TFT FeatherWing V2 - 3.5" 480x320 Touchscreen for Feathers - TSC2007 I2C 0x48 Touch Controller](https://exp-tech.de/en/products/adafruit-tft-featherwing-35-480x320-touchscreen-for-feathers?variant=47546750796103)

I suspect the reason for that (just my best guess) is the TSC2007 Touch controller because on the installer page (6.4 or so) it says STMPE610 under Touch Controller. I already found out that there is a nightly installer, but also the TSC2007 or v2 doesn't show up here. When using a FW-File from the Actions-Tab on Github, i.e. huzzah32-featherwing-35-v2_ota_v0.7.0-rc12_944f92e.bin, the screen won't show anything anymore.

But what confuses me, is that it shows up as supported inside the Documentation: https://www.openhasp.com/0.7.0/hardware/adafruit/featherwing/#supported-devices:~:text=(HX8357D%20with%20TSC2007%20resistive%20touch)

Any ideas? Would I need to manually compile a FW for the FeatherWing V2 feather, and if yes, would you mind pointing me in the right direction? I currently have little to non experience with compiling FW 😅

**Comment by CHristapherenese** [comment]
Did you get this working, I am having a very similar problem with compatibility.
